.github/workflows/coverage.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         with:
           persist-credentials: false
 
-      - uses: actions-rs/toolchain@v1
+      - uses: actions-rs/toolchain@v1.0.7
         with:
           toolchain: nightly
```

Comment:
Do we still need the nightly toolchain?

Reply:
Yes, the source-based coverage support in Rust/llvm is still only available on the nightly channel AFAIK, but also specifically `cargo-llvm-cov` requires specific unstable nightly flags:

![image](https://user-images.githubusercontent.com/552961/121938733-2d6e9e00-cd1a-11eb-9621-8a67b88fc5a4.png)